include/curl/easy.h

```diff
@@ -25,6 +25,92 @@
 extern "C" {
 #endif
 
+/* Application can use a memory PKCS12 certificate with CURLOPT_SSLCERT
+ *  with OpenSSL, Schannel or SecTransport
+ *
+ *
+ *  Example if with certificate binary data and size in:
+ *    void* certdata;
+ *    size_t certsize;
+ *    ...
+ *  set the certificate with:
+ *    struct curl_blob structblob;
+ *    curl_init_blob_dup(&structblob, certdata, certsize);
+ *    curl_easy_setopt(curl, CURLOPT_KEYPASSWD, &structblob);
+ *             my_setopt_str(curl, CURLOPT_SSLCERT, &structblob);
+ *  struct curl_blob is just a 0x20 bytes structures which will
```

Comment:
You don't know how big the struct is in bytes because this comment doesn't know the size of `size_t` for example. Don't assume!

Reply:
I remove this comment about size which was true in previous version